utils/utils.go

```diff
@@ -5,16 +5,12 @@ import (
 	"errors"
 	"fmt"
 	"math/big"
-	"strings"
 
 	"github.com/openrelayxyz/xplugeth"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/node"
-
-	"github.com/Shopify/sarama"
-	"github.com/openrelayxyz/cardinal-streams/transports"
 )
```

Comment:
So these imports caused an unexpected result. If we were to try and get a foundation build, the foundation core package hooks filed uses the getTD function and thus imports this package. The result is that the imports here were not present and cause the test to fail. By placing the imports, and the commiserate functions in another package we avoid this problem.